### ISSUE TITLE
PurchaseStateAndroid Enum values are not correct 

### ISSUE THREAD

**Issue by s123121** [dooboolab/react-native-iap#839]
### Version of react-native-iap
4.2.0

### Version of react-native
0.60.5

### Platforms you faced the error (IOS or Android or both?)
Android

### Expected behavior
I can call the function `finishTransaction` successfully, however everytime i called the function, the error `purchase is not suitable to be purchased` always show up

### Tested environment (Emulator? Real Device?)
Emulator

### Steps to reproduce the behavior
I believe the error is introduced in this commit https://github.com/dooboolab/react-native-iap/pull/833/commits/e7eff656d490eb44f825f4e4a38a0d25b7c22c27
```
export enum PurchaseStateAndroid {
  Purchased = 0,
  Canceled = 1,
  Pending = 2,
}
```
According to my research, the values here are for old billing flow. You can see the new values here: 
https://developer.android.com/reference/com/android/billingclient/api/Purchase.PurchaseState.html

```
export enum PurchaseStateAndroid {
  PENDING = 0,
  PURCHASED = 1,
  UNSPECIFIED_STATE = 2,
}
```

**Comment by hyochan** [comment]
:scream: Would you like to give a `PR` for this?

**Comment by hyochan** [comment]
Actually I'll fix this.

**Comment by s123121** [comment]
I am not familiar with the billing on Android. However, I believe the old billing `purchaseState` is not consistent with the new `purchaseState` value (https://developer.android.com/google/play/billing/billing_reference.html). 
Do we need to insert condition for those who use the old billing system?

**Comment by photoroulette** [comment]
We are seeing most purchases on Android getting refunded after migrating to react-native-iap@4.2.2 and the finishTransaction method. Not 100% sure it is related to this change, but could it be?

**Comment by DanielHarold** [comment]
Hello, according to https://developer.android.com/reference/com/android/billingclient/api/Purchase.PurchaseState.html (cited above), `PENDING` appears to be 2 and `UNSPECIFIED_STATE` appears to be 0. Notice that the constants are listed in descending order on that page. So `PENDING` and `UNSPECIFIED_STATE` may need to be switched?

**Comment by hyochan** [comment]
@DanielHarold You are right. I'm not quite sure when it has changed. I'll fix it now.

**Comment by hyochan** [comment]
[Fixed](https://github.com/dooboolab/react-native-iap/commit/fb8088309f98bca77872ac1f169129ba4f21ce60).

Released in `4.3.3`

**Comment by DanielHarold** [comment]
Wonderful, thank you!
